### DISCUSSION TITLE
Select inside x-if resets values

### DISCUSSION THREAD

**Discussion by murwa** [alpinejs/alpine#3645] [7. Bugs (archived)]
I've made a pen [here](https://codepen.io/Murwa/pen/KKrXmzG) to show this behaviour. To reproduce on the pen:-
- select a value
- press next
- press previous to show the select input again

The initially selected framework will not be selected on the select, but the value is still intact. It's like x-model just turns a blind eye.

**Comment by ekwoka** [comment]
just add `:selected="option === framework` on the option
